pkg/cache/placeholder_manager_test.go

```diff
@@ -261,3 +261,24 @@ func TestPlaceholderManagerStartStop(t *testing.T) {
 	time.Sleep(5 * time.Millisecond)
 	assert.Equal(t, mgr.isRunning(), false, "placeholder manager has not stopped")
 }
+
+func TestPlaceholderManagerCleanup(t *testing.T) {
+	mockedAPIProvider := client.NewMockedAPIProvider()
+	pod1 := &v1.Pod{
+		TypeMeta: apis.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: apis.ObjectMeta{
+			Name: "pod-01",
+			UID:  "UID-01",
+		},
+	}
+	mgr := NewPlaceholderManager(mockedAPIProvider.GetAPIs())
+	mgr.Start()
+	assert.Equal(t, mgr.isRunning(), true, "manager should be running after start")
+	mgr.orphanPods["task01"] = pod1
+	assert.Equal(t, len(mgr.orphanPods), 1)
+	<-time.After(5 * time.Second)
```

Comment:
we can't wait for 5s in UT for this single test
what we can do here is to add a function that can change the default 5s interval for the placeholder cleanup, e.g `setCleanupInterval(interval time.Duration)`. During this test, set the interval to 100millsecond, etc. And remember to set it back to 5s after the test.

Reply:
![image](https://user-images.githubusercontent.com/48027290/133586284-26abf199-96c1-44dd-896d-90a20e4dcbef.png)